release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+==== Complete Upstreaming of RBAC, Then Downstreaming it Back into OpenShift
+
+The rolebinding and RBAC experience is now the same across all Kubernetes
+distributions.
+
+Administrators do not have to do anything for this migration to occur. The
+upgrade process to {product-title} 3.7 offers a seamless experience. Now, the
+user experience is consistent with upstream.
+
+A role can be defined within a namespace with a `Role`, or cluster-wide with a
+`ClusterRole`.
+
+A `RoleBinding` or `ClusterRoleBinding` binds a role to subjects. Subjects can
+be groups, users, or service accounts. A role binding grants the permissions
+defined in a role.
+
+[[ocp-37-flexvolume-support-for-non-stotage-use-cases]]
+==== Official FlexVolune Support for Non-storage Use Cases
+
+There is now a supported interface to allow you to bind and mount in content
+from a running pod. FlexVolume is a script interface that runs on the kublet and
+offers five main functions to help you mount in content such as device drivers,
+secrets, and certificates as bind mounts to the container from the host:
+
+* `init` - Initialize the volume driver.
+* `attach` - Attach the volume to the host.
+* `mount` - Mount the volume on the host. This is the part that makes the volume available
+to the host to mount it in *_/var/lib/kubelet_*.
+* `unmount` - Unmount the volume.
+* `detach` - Detach the volume from the host.
+
+[[ocp-37-longer-lived-api-tokens-to-oauth-clients]]
+==== Issue Longer-lived API Tokens to OAuth Clients
+
+Administrators now have the ability to set different token timeouts for the
+different ways users connect to {product-title} (for example, via the `oc` command
+line, from a GitHub authentication, or from the web console).
+
+Administrators can edit `oauthclients` and set the `accessTokenMaxAgeSeconds` to
+a time value in seconds that meets their needs.
+
+There are three possible OAuth client types:
+
+. `openshift-web-console` -  The client used to request tokens for the OpenShift web console.
+
+. `openshift-browser-client` - The client used to request tokens at
+*_/oauth/token/request_* with a user-agent that can handle interactive logins,
+such as using Auth from GitHub, Google Authenticator, and so on.
+
+. `openshift-challenging-client` - The client used to request tokens with a user-agent that can
+   handle WWW-Authenticate challenges, such as the `oc` command line.
+
+- When `accessTokenMaxAgeSeconds` is set to `0`, tokens do not expire.
+- When left blank, {product-title} uses the definition in `master-config`.
+- Edit the client of interest via:
++
+----
+# oc edit oauthclients openshift-browser-client
+----
+
+- Set `accessTokenMaxAgeSeconds` to `600`.
+- Check the setting via:
++
+----
+# oc get oauthaccesstoken
+----
+
+See
+xref:../architecture/additional_concepts/other_api_objects.adoc#accessTokenMaxAgeSeconds[Other
+API Objects] for more information.
+
+[[ocp-37-storage]]
+=== Storage
+
+[[ocp-37-local-persistent-volumes]]
```

Comment:
Missing a note (like what we had in 3.6): This feature is currently in Technology Preview and not for production workloads.